### ISSUE TITLE
Crash on deleting the last cloud layer

### ISSUE THREAD

**Issue by shun-iwasawa** [opentoonz/kumoworks#5]
[Posted by Rodney in OpenToonz Users Forum](https://groups.google.com/d/msg/opentoonz_en/AuwtVmh4dQw/rI3MRH1TCgAJ)

### Issue Summary

Rodney [commented](https://groups.google.com/d/msg/opentoonz_en/AuwtVmh4dQw/rI3MRH1TCgAJ):
>If there is only one Cloud Layer (say we create many but then remove all but one) and then delete the only remaining cloud layer... Kumoworks will crash.

Jane [commented](https://groups.google.com/d/msg/opentoonz_en/AuwtVmh4dQw/jmof0356CgAJ)
>I can't delete the only remaining cloud layer, so I don't get a crash (macOS). 
>On the UI side, changes to the workspace don't save. Also, the delete button (for the cloud layers) is hidden (at least with the default window size). 
![image](https://user-images.githubusercontent.com/17974955/56478438-628c5680-64ea-11e9-93ef-2b46c52ddfe4.png)

**Comment by shun-iwasawa** [comment]
@RodneyBaker
As Jane wrote, I think I made the Remove button to be disabled when the amount of cloud layer is one. I may miss something.
Can you please let me know how to reproduce the problem (i.e. to delete the last cloud layer)? Thanks!

**Comment by RodneyBaker** [comment]
@shun-iwasawa  Thanks for the follow up.
Interestingly, I currently can't reproduce the problem of deleting the last cloud layer.

I've tried every possible way I can think of and as you've suggested the 'remove' option is grayed out and therefore cannot be used.  As such, I'm not sure what I did to be able to produce the problem in the earlier situation.

Recommend closing.
I will add commentary (or open a more detailed issue) if/when I can reproduce.

Thanks!
